tasks/main.yml

```diff
@@ -7,6 +7,11 @@
 - name: Include OS variables
   include_vars: "{{ ansible_os_family }}.yml"
 
+- name: Set nomad_servers to empty list rather than the default macro if we are using consul
```

Comment:
simplest way is just adding een `if` `else` statement :)

Another thing you could do is just overwrite the `nomad_servers`, this is the reason the macro is in the defaults file file and not in the vars file in the first place. (I did think about special use cases when I wrote that code)